### DISCUSSION TITLE
Request a launcher tool for MacOS

### DISCUSSION THREAD

**Discussion by seyedrezabazyar** [Ulauncher/Ulauncher#1016] [General]
Hello

I need this tool for Mac OS. Can I run it on a Mac? Or is there an alternative tool for Mac?

**Comment by friday** [comment]
https://www.slant.co/topics/1819/~app-launchers-for-mac-os-x

**Comment by brpaz** [comment]
@seyedrezabazyar I recommend [Raycast](https://www.raycast.com/). It has tons of great extensions similar to Ulauncher but is a lot more powerful in some parts. I would love that Ulauncher took some inspiration in the way they handle actions and support features like forms and detail views.
